tailwind.config.js

```diff
@@ -28,17 +28,42 @@ module.exports = {
           "base-200": "#f1f5f9",
           "base-300": "#e2e8f0",
 
-          primary: "#4885ed",
-          "primary-content": "#E8F0FE",
-          secondary: "#3cba54",
-          accent: "#f4c20d",
-          neutral: "#0f172a",
-          "neutral-content":"#AEB0B3",
+          primary: "#4285F4",
+          ".primary-fade": {
+            "background-color": "#8AB4F8",
+          },
+          ".primary-focus": {
+            "background-color": "#1A73E8",
+          },
+          secondary: "#24C1E0",
+          ".secondary-fade": {
+            "background-color": "#78D9EC",
+          },
+          ".secondary-focus": {
+            "background-color": "#12B5CB",
+          },
+          accent: "#D93025",
+          "accent-fade": {
+            "background-color": "#FCE8E6",
+          },
+          neutral: "#E8F0FE",
 
-          info: "#3b82f6",
-          success: "#22c55e",
-          warning: "#eab308",
-          error: "#ef4444",
+          info: "#1A73E8",
+          "info-fade": {
+            "background-color": "#E8F0FE",
+          },
+          success: "#1E8E3E",
+          "success-fade": {
+            "background-color": "#E6F4EA",
+          },
+          warning: "#F9Ab00",
+          "warning-fade": {
+            "background-color": "#FEF7E0",
+          },
+          error: "#D93025",
+          "error-fade": {
+            "background-color": "#FCE8E6",
+          },
         },
       },
       {
```

Comment:
For now we can do just the `light` theme, but we will need to do `dark` theme soon